tools/gherkinize.py

```diff
@@ -764,14 +855,12 @@ def __init__(
         logger.debug(f"Scenario {source.name}")
         self.name = source.name
         self.description = source.description
-        self.tag = (
-            "@wip\n"
-            if skip_list.should_skip(feature.name, section.name, source.name)
-            else ""
-        )
-        self.preconditions: List[str] = []
-        self.events: List[str] = []
-        self.outcomes: List[str] = []
+        self.tag = ""
+        for tag in wip_config.tags_for(feature.name, section.name, source.name):
+            self.tag += f"{tag}\n"
```

Comment:
1. Isn't this `self.tag = "\n".join(wip_config.tags_for(feature.name, section.name, source.name)`?
2. Where'd the '@' go? Is it in the JINJA template? Or somewhere else?

Reply:
1. Cleaned this up to do the looping in the template, which I should have done all along.
2. The `@` is always carried along in the "tag" value. I did this so that Gherkin tags would always _look like_ Gherkin tags.